tempMonApp/src/tempMon.c

```diff
@@ -405,3 +410,57 @@ cpu_thermal_init(int hwmon_inst, struct temp_state* state)
   state->fd = open(file, O_RDONLY);
   return state->fd < 0 ? -1 : 0;
 }
+
+/******************************************************************************
+ * IOC shell function registration
+ *****************************************************************************/
+
+static void
+hwmonStatusCallFunc(const iocshArgBuf* args)
+{
+  /** Caller wants us to re-index all hwmon types */
+  if (args[0].ival != 0) {
+    printf("Available hwmon sensors:\n");
+
+    /** Loop through all hwmon types and read their type */
+    for (int i = 0; i < 99; ++i) {
```

Comment:
Why 99? Can this be a macro?

Reply:
Yeah it can be a macro; the 99 is just a guard for infinite loop. I guess it doesn't really matter if I put `while(true)` there though.